app/component/itinerary/Navigator.js

```diff
@@ -33,6 +34,19 @@ function Navigator({
 
   return (
     <div>
+      <div className="navigator-top-section">
+        <button
+          type="button"
+          onClick={() => {
+            if (window && window.location) {
```

Comment:
Do no reload page.  It is slow and not necessary. 

Navigator visibility is controlled by itinerary page state and can be changed using the function prop.

Reply:
Give me here example how we do this, please? I have a try to do this same way than we did at NavigatorButton, but it didn't work.

Reply:
Can you check this functionality? I made little change to it.